spec/models/queue_item_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper.rb'
+
+describe QueueItem do
+  it { should belong_to :video }
+  it { should belong_to :user }
+  it { should validate_presence_of :video }
+  it { should validate_presence_of :user }
+
+  let(:user) { Fabricate(:user) }
+
+  describe '#video_title' do
```

Comment:
You can possibly refactor even these specs - `#video_title`, `#category_name`, `#category` - with should matchers - check `delegate_method` - https://github.com/thoughtbot/shoulda-matchers/blob/master/lib/shoulda/matchers/independent/delegate_method_matcher.rb